### ISSUE TITLE
MAS Accessibility Testing Bugs

### ISSUE THREAD

**Issue by grubioe** [dotnet/wpf#1350]
Catch all Issue to Track MAS-Accessibility bugs for .NET Core 3 Sample Apps. 

List can be found via [AzDO query](https://devdiv.visualstudio.com/DevDiv/_queries/query/?tempQueryId=0ab38350-4eee-4cbe-9440-bc048b5d6265)

ID | Title
-- | --
947059 | A11y_.NETCore3.0(WPFandASP.NETCore)_WPFBrush_SolidColorBrush-Animation_Keyboard:   Inactive element receives keyboard focus while navigation.
947071 | A11y_.NETCore3.0(WPFandASP.NETCore)_WPFBrush_SolidColorBrush-Animation_ScreenReader:   Identical names available for buttons as “Start Animation” in the main   content.
947087 | A11y_.NETCore3.0(WPFandASP.NETCore)_WPFBrush_SolidColorBrush-Animation_ScreenReader:   Using NVDA the content available in the main content is not announced to the   screen reader users.
947098 | A11y_.NETCore3.0(WPFandASP.NETCore)_WPFBrush_PredefinedBrushes_ScreenReader:   The "Predefined Brushes" text is not announced as a header element   for the screen reader users.
947717 | A11y_.NETCore3.0(WPFandASP.NETCore)_WPFBrush_Contents_ScreenReader:   Using narrator the decorative images available for   the"Introduction", "Syntax", etc. are announced as   "image" to the screen reader users.
947720 | A11y_.NETCore3.0(WPFandASP.NETCore)_WPFBrush_PredefinedBrushes_ScreenReader:   Using NVDA the entire content available in the main window is not announced   to the screen reader users.
947804 | A11y_.NETCore3.0(WPFandASP.NETCore)_WPFBrush_Contents_ScreenReader:   Inappropriate name announced for the treeview items of the   "Contents" for the screen reader users while navigation.
947810 | A11y_.NETCore3.0(WPFandASP.NETCore)_WPFBrush_Contents_Keyboard:   While user navigates between the treeview items of the "Contents"   the content within the main window changes.
947814 | A11y_.NETCore3.0(WPFandASP.NETCore)_WPFBrush_Contents_ScreenReader:   Using narrator the proper role as "treeview item" is not announced   for options of "Contents" for screen reader users.
948205 | A11y_.NETCore3.0(WPFandASP.NETCore)_WPFBrush_Contents_Keyboard:   The treeview items with toggle button receive keyboard focus twice while   navigating using the arrow key.
948255 | A11y_.NETCore3.0(WPFandASP.NETCore)_WPFBrush_Contents_ScreenReader:   Using narrator the expand/collapse state not announced for the "Contents"   button for screen reader users.der user.
948316 | A11y_.NETCore3.0(WPFandASP.NETCore)_WPFBrush_Introduction_ScreenReader:   The informative images present in the main window are not provided with   descriptive text.
948963 | A11y_.NETCore3.0(WPFandASP.NETCore)_WPFBrush_Contents_AI4D:   (Brushes/button '') The Name property of a focusable element must not be   null.
948970 | A11y_.NETCore3.0(WPFandASP.NETCore)_WPFBrush_LinearGradient-InteractiveExample_AI4D:   (Brushes/combo box '') The Name property of a focusable element must not be   null.
949047 | A11y_.NETCore3.0(WPFandASP.NETCore)_WPFBrush_LinearGradient-InteractiveExample_AI4D:   (Brushes/button '') An onscreen element must not have a null   BoundingRectangle property.
949491 | A11y_.NETCore3.0(WPFandASP.NETCore)_WPFBrush_LinearGradient-InteractiveExample_ScreenReader:   The sliders present in the main window are not actionable using keyboard and   narrator.
949578 | A11y_.NETCore3.0(WPFandASP.NETCore)_WPFBrush_LinearGradient-InteractiveExample_ScreenReader:   Using narrator some inappropriate information is announced for the edit field   for the screen reader users.
949765 | A11y_.NETCore3.0(WPFandASP.NETCore)_WPF   Visibility Changes_HighContrast: Screen content is not visible when we apply   High contrast Black theme.
949977 | A11y_.NETCore3.0(WPFandASP.NETCore)_WPF   Text Formatting_HighContrast: Button selection is not visible when we apply   High contrast Black theme.
950655 | A11y_.NETCore3.0(WPFandASP.NETCore)_   DataTemplatingIntro_ScreenReader: When control goes on to the information   field using tab key screen reader (nvda + narrator) does not announce   anything.
950658 | A11y_.NETCore3.0(WPFandASP.NETCore)_   DataTemplatingIntro_ScreenReader: Screen reader (nvda + narrator ) does not   announce the description, priority text of the 'groceries, laundry, clean,   proposals' fields.
950660 | A11y_.NETCore3.0(WPFandASP.NETCore)_   DataTemplatingIntro_ScreenReader: Screen reader (nvda + narrator ) does not   announce the label 'get ready for family reunion' available on the get ready   for family reunion field.
951013 | A11y_.NETCore3.0(WPFandASP.NETCore)_WPF   Message Box_ScreenReader: Screen reader is not announcing Expanded State for   combobox.
951016 | A11y_.NETCore3.0(WPFandASP.NETCore)_AlternatingAppearanceOfItems_ScreenReader:   Screen reader (nvda + narrator) does not announce the list item names.
951069 | A11y_.NETCore3.0(WPFandASP.NETCore)_WPF   Flow Document Properties_Left Pane_HighContrast: Keyboard focus in not   visible in high contrast White theme and Normal mode.
951115 | A11y_.NETCore3.0(WPFandASP.NETCore)_WPFUsingElements_ClearControls_Keyboard:   The keybaord users are unable to activate the "Clear Controls"   tabitem using "Enter" key.
951117 | A11y_.NETCore3.0(WPFandASP.NETCore)_WPFUsingElements_ClearControls_ScreenReader:   The "UI Element Collection - Methods" text is not announced as a   header element for the screen reader users.
951122 | A11y_.NETCore3.0(WPFandASP.NETCore)_WPFUsingElements_InsertControl_ScreenReader:   Identical names available for multiple buttons in the main content.

**Comment by rladuca** [comment]
IMO, a catch all does not seem appropriate for these.  Each bug is going to have its own PR and progress should be tracked by issues opening and closing.  External contributors/customers will not be able to easily track and understand these issues.

These sorts of merged bugs might be fine for test issue investigations because that code is not product code and has no external effect.  Even in that case, if we find a test issue is due to a product bug there is a separate issue filed to track.

I would very much prefer that if we fix one of these bugs, it be its own separate issue with all appropriate information filed in it.  I'm inclined to not approve any PR against this bundled issue.
